nengo_loihi/builder.py

```diff
@@ -760,7 +761,8 @@ def conn_probe(model, probe):
             else:
                 input_dim = len(func[0])
         transform = kwargs['transform']
-        transform = np.asarray(transform, dtype=np.float64)
+        assert isinstance(transform, Dense)
```

Comment:
I'd also do a BuildError here (I'm looking through this backwards :back:)

Reply:
Same as above.